### DISCUSSION TITLE
Category Feature Request

### DISCUSSION THREAD

**Discussion by dragow0lf** [xgi/houdoku#115] [Feature Requests]
Basically let us make lists, Would help us sort our manga, You could make premade categories (Reading, Plan To Read, Completed) or you could let us make our own lists. I personally would prefer our own lists, it would also let us hide stuff we are reading. Basically something like aniyomi's category system

**Comment by xgi** [comment]
Does the "User Tags" feature work for this? On a series page there's a "User Tags" field where you can enter give a series custom tags, and then on the library page you can click "Filter Tags" at the top to show series with that tag.

The controls are a bit janky but I think it gives the functionality you're describing.

**Comment by dragow0lf** [comment]
Yea it gives the functionality, but definitely janky. Would be nice to let us directly add the manga to the usertag instead of first adding it to the library and then giving it the tag. Also, would also be great if we could exclude a certain tag using filter tag instead of just showing a particular tag using filter tag.
